docs/source/CONTRIBUTING.md

```diff
@@ -1,138 +1,157 @@
-  important to get this change landed. Remember that you're asking for valuable
-  time from other professional developers.
+- Review other people's changes. If you help out, others will be more willing to do the same for you. Good will is our
+  currency.
+- Split your change into multiple smaller changes. The smaller your change, the higher the probability that somebody
+  will take a quick look at it.
+- Ping the change on [slack]. If it is urgent, provide reasons why it is important to get this change landed. Remember
+  that you're asking for valuable time from other professional developers.
 
-**Note that anyone is welcome to review and give feedback on a change, but only
-people with commit access to the repository can approve it.**
+**Note that anyone is welcome to review and give feedback on a change, but only people with commit access to the
+repository can approve it.**
 
 ## Attribution of Changes
 
-When contributors submit a change to this project, after that change is approved,
-other developers with commit access may commit it for the author. When doing so,
-it is important to retain correct attribution of the contribution. Generally
-speaking, Git handles attribution automatically.
+When contributors submit a change to this project, after that change is approved, other developers with commit access
+may commit it for the author. When doing so, it is important to retain correct attribution of the contribution.
+Generally speaking, Git handles attribution automatically.
 
 ## Writing Code
 
 ### Setting Up A Development Environment
 
-This project uses [Poetry] to manage its dependencies and do a lot of the heavy
-lifting. This includes managing development environments! If you are not
-familiar with this tool, we highly recommend checking out [their docs][poetry docs]
+This project uses [Poetry] to manage its dependencies and do a lot of the heavy lifting. This includes managing
+development environments! If you are not familiar with this tool, we highly recommend checking
+out [their docs][poetry docs]
 to get used to the basic usage.
 
-Now, setting up a development environment is super simple! Additional info if you run into trouble: [Poetry Environments]
+Now, setting up a development environment is super simple! Additional info if you run into
+trouble: [Poetry Environments]
 
 Step 1: [Install Poetry]  
 Step 2: Run ``poetry install``  
-Step 3: Optionally Run ``poetry shell``  
+Step 3: Optionally Run ``poetry shell``
 
 Done!
 
 ### Code Style
 
-To make code formatting easy on developers, and to simplify the conversation
-around pull request reviews, this project has adopted the
-[black] code formatter. This formatter must be run against any new code written
-for this project. The advantage is, you no longer have to think about how your
-code is styled; it's all handled for you!
+From [PEP 8 -- Style Guide for Python Code][PEP 8#a-foolish-consistency-is-the-hobgoblin-of-little-minds]
```

Comment:
This link doesn't appear to be working the way it was intended. I checked both here in the GitHub preview, and in my VSCode preview.
```suggestion
From [PEP 8 -- Style Guide for Python Code][PEP 8]
```

Reply:
You will also need to change line 250 to:

```markdown
[PEP 8]: https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds
```
It appears that particular link is no longer being used in this document, so it can be repurposed for this.